pyked/chemked.py

```diff
@@ -685,7 +688,10 @@ def __init__(self, properties):
                     values = np.array(hist['values'])
                 else:
                     # Load the values from a file
-                    values = np.genfromtxt(hist['values']['filename'], delimiter=',')
+                    filename = hist['values']['filename']
+                    if not isabs(filename):
+                        filename = join(directory, filename)
```

Comment:
I think this will fail if the input is a `dict` and therefore `directory` is `None`

Reply:
Changing the default directory to `''` would fix that. But, if the input is a dict, then the filename must be specified as an absolute path, right? Since there's no yaml file, a path relative to a yaml file wouldn't make sense. So, line 693 won't be run anyway.

Reply:
I don't think that change will fix the problem on this line, because the `directory` argument gets set to `None` if `yaml_file` is `None`, so this line will join `None` and `filename`, which won't work. Actually, I think that if people provide a dictionary input, the CSV file has to be specified relative to the PWD of the Python process, or as an absolute file name, but we don't need to check for that case.

Reply:
Good catch. I just pushed another commit so that `directory` will never be set to `None`. Now, if `yaml_file` is `None`, then `directory` will be `''`